drivers/hud_pit/app/models/hud_pit/generators/pit/fy2024/adults.rb

```diff
@@ -0,0 +1,88 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudPit::Generators::Pit::Fy2024
+  class Adults < Base
+    QUESTION_NUMBER = 'Households without Children'.freeze
```

Comment:
This is listed as `Adult Only (without children)` in the HDX, may not be worth changing if difficult.